src/ensembl/src/content/app/entity-viewer/EntityViewer.tsx

```diff
@@ -106,7 +109,8 @@ const mapStateToProps = (state: RootState) => {
 const mapDispatchToProps = {
   setDataFromUrl,
   fetchGenomeData,
-  toggleSidebar
+  toggleSidebar,
+  updateEnsObject
```

Comment:
does `updateEnsObject ` do anything?

Reply:
Ah that's a remnant from the merge with dev. I'll remove it.

Reply:
✔️